common/src/IO/SystemPaths.cpp

```diff
@@ -22,25 +22,33 @@
 #include "IO/DiskFileSystem.h"
 #include "IO/Path.h"
 
-#include <wx/stdpaths.h>
+#include <iostream>
 
-#if defined __APPLE__
-#include "CoreFoundation/CoreFoundation.h"
-#elif defined _WIN32
-#include <Windows.h>
-#elif defined __WXGTK20__
-#include <unistd.h>
-#endif
+#include <wx/stdpaths.h>
+#include <wx/utils.h>
 
 namespace TrenchBroom {
     namespace IO {
         namespace SystemPaths {
             Path appDirectory() {
-                const IO::Path executablePath(wxStandardPaths::Get().GetExecutablePath().ToStdString());
-                return executablePath.deleteLastComponent();
+                return IO::Path(wxStandardPaths::Get().GetExecutablePath().ToStdString()).deleteLastComponent();
+            }
+
+#if defined __linux__
+            static bool getDevMode() {
+                wxString value;
+                if (!wxGetEnv("TB_DEV_MODE", &value))
+                    return false;
+                return value != "0";
             }
+#endif
             
             Path resourceDirectory() {
+#if defined __linux__
+                static const bool DevMode = getDevMode();
```

Comment:
I'm not sure if it's safe to call `wxGetEnv` and use `wxString` from a static initializer.
More cumbersome but safer:
```
static bool CheckedDevMode;
static bool DevMode;
if (!CheckedDevMode) {
    CheckedDevMode = true;
    DevMode = getDevMode();
}
```